src_assets/common/assets/web/index.html

```diff
@@ -107,21 +107,27 @@ <h2>Legal</h2>
         let v = this.githubVersion.name;
         // If the version starts with a v, remove it
         if (v.indexOf("v") === 0) v = v.substring(1);
+
+        // if nightly, we do an additional check to make sure its an actual upgrade.
+        if (this.version.split('.').length > 3) {
```

Comment:
I'd prefer this to be just set to `=== 4`. If it's more than 4 it's for sure not a nightly.

Or set all the `=== 4` to `> 3`.